server/neptune/workflows/internal/deploy/terraform/runner.go

```diff
@@ -31,6 +33,11 @@ func (r *WorkflowRunner) Run(ctx workflow.Context, deploymentInfo DeploymentInfo
 	id := deploymentInfo.ID
 	ctx = workflow.WithChildOptions(ctx, workflow.ChildWorkflowOptions{
 		WorkflowID: id.String(),
+		// we shouldn't ever percolate failures up unless they are user errors (aka. Terraform specific)
+		// retrying indefinitely allows us to fix whatever issue that comes about without involving the user to redeploy
+		RetryPolicy: &temporal.RetryPolicy{
+			NonRetryableErrorTypes: []string{"TerraformClientError", "PlanRejectedError"},
```

Comment:
I haven't looked to closely at the SDK code, but by default Workflow Executions don't retry: https://docs.temporal.io/application-development/features#workflow-retries, do you need to specify the other RetryPolicy parameters to create the full policy?